tests/runtime/call

```diff
@@ -316,6 +316,14 @@ REQUIRES_FEATURE dpath
 TIMEOUT 5
 AFTER ./testprogs/syscall read
 
+NAME path_invalid_helper
+PROG kfunc:vfs_read { print(path(args->file->f_path)); }
+EXPECT \nstdin:1:18-48: ERROR: helper bpf_d_path not supported in probe\n
+REQUIRES_FEATURE dpath
```

Comment:
I'm confused about this test. If `REQUIRES_FEATURE dpath` condition is met, then it means dpath is supported and loads correctly (see `BPFfeature::has_d_path()`). I suspect the CI is passing b/c the condition is not met so the test is noop. But on a machine with dpath, this test will fail, right?

Reply:
The program is rejected by verifier because it uses the `path` helper in a kfunc not on the list of functions where `bpf_d_path` is allowed (see `btf_allowlist_d_path` in `kernel/trace/bpf_trace.c`). On systems without `d_path` support, the test is not run, because `d_path` always fails there, hence it's pointless.

Reply:
Ah, I see. I thought you were testing that path() was not supported on the machine at all.

Mind changing the name of the test case to something like "use_path_in_unsupported_function" ?

Reply:
I renamed it to `path_in_unsupported_kfunc`.